ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -77,6 +77,11 @@ func (scbuilder *SiteConfigBuilder) getClusterCRs(clusterId int, siteConfigTemp
 	for _, cr := range scbuilder.SourceClusterCRs {
 		mapSourceCR := cr.(map[string]interface{})
 
+		// If cluster has not set a clusterImageSetNameRef we use the site one
+		if siteConfigTemp.Spec.Clusters[clusterId].ClusterImageSetNameRef == "" {
```

Comment:
Would u move this to the Build func (similar as networkType) AND add validation if both spec->clusterImageSetNameRef AND spec->clusters[i]->clusterImageSetNameRef are empty

Reply:
Done.